invesalius/gui/task_navigator.py

```diff
@@ -1065,27 +1072,101 @@ def __init__(self, parent, nav_hub):
         self.__bind_events()
 
     def __bind_events(self):
-        Publisher.subscribe(self.OnObjectUpdate, "Update object registration")
+        pass
+
+    def onRecord(self, evt):
```

Comment:
Could you change the name of this function to be more explicit, please? Like OnRecordStylusOrientation

Reply:
This and the above 2 comments are resolved in commit 81f4251c7f3e82200dce83ab55133b7d7546d2e9